docs/WRAPPER_ZUSTAND_PERSIST_MIDDLEWARE.md

```diff
@@ -11,20 +11,14 @@ const storage = new MMKV()
 
 const zustandStorage: StateStorage = {
   setItem: (name, value) => {
-    storage.set(name, value)
-    return Promise.resolve()
+    return storage.set(name, value)
   },
   getItem: (name) => {
     const value = storage.getString(name)
-    if (value) {
-      return Promise.resolve(value)
-    } else {
-      return null
-    }
+    return value || null
```

Comment:
```suggestion
    return value ?? null
```

Reply:
@mrousavy if it returns an empty string then it will be an empty string instead of null.
```ts
'' ?? null // returns ''
'' || null // returns null
```

Reply:
Yes empty string is desired here because it has a value - null is when it explicitly does not have a value

Reply:
@mrousavy it's done! I though that getString will always return a string, somehow without checking types :))